mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala

```diff
@@ -158,6 +158,9 @@ class Word2Vec extends Serializable with Logging {
       .sortWith((a, b) => a.cn > b.cn)
     
     vocabSize = vocab.length
+    require(vocabSize > 0, "The vocabulary size should be large than 0. You may need to check " +
```

Comment:
Sounds good.  @yinxusen Sorry for asking you to revert your change, but can you please stick with "require" per @srowen 's advice?  Thank you!